lisk/src/index.js

```diff
@@ -118,6 +118,6 @@ try {
 			process.exit();
 		});
 } catch (e) {
-	console.error('Application start error.', e.errors);
+	console.error('Application start error.', e);
```

Comment:
Why this change? Just curious

Reply:
Ah this shouldnt be commited (not related to this PR)
but it shows undefined, because this error doesn't have `errors` property.
It will be fixed in other PR